cmd/gen-go-sample/testdata/sample_unary.want

```diff
@@ -28,6 +28,14 @@ foopb "path.to/pb/foo"
 
 // [START awesome_region]
 
+// Construct a complex request object,
```

Comment:
Though I get the problem with changing the verb mood, one thing you could do is go half-way to the Go standard and include the `functionName + ": " + description`, so that this reads, for example,

```
// sampleUnaryMethod: Construct a complex request object
```

or **even better**, `functionName + " will " + lowerFirstChar(description)`:

```
// sampleUndaryMethod will construct a complex request object
```

I _think_ the latter we can do safely and always grammatically (assuming that the verb is always imperative as in this example ["Construct ..."] and not declarative [ie "Constructs ..." would break this]), and that ought to be sure to shut up `gofmt`. The former may or may not shut up `gofmt`.

Is there a standard/prevalence of imperative vs declarative in proto comments?

Reply:
There are no standard for proto comments AFAIK. AIP however, has requirements (https://aip.dev/192#style).

I am onboard with function name + : + rest of comments. I am a bit reluctant to do function name + "will" + lowerFirstChar(description) because we are assuming the description starts with a verb and the verb is not in third-person singular form, the latter especially, because that style is fairly common (at least AIP and Java are two that I know of).

Reply:
Makes sense. The `:` solution sounds like the best compromise between what people might write in their protos and what Go idiomaticity requires.